8dev_restapi.js

```diff
@@ -120,20 +121,46 @@ class Service extends EventEmitter {
     this.client = new rest.Client();
     this.endpoints = [];
     this.addTlvSerializer();
+    this.express = express();
+    this.express.use(parser.json());
   }
 
-  start(interval = 1234) {
-    this.pollTimer = setInterval(() => {
-      this.get('/notification/pull').then((dataAndResponse) => {
-        this._processEvents(dataAndResponse.data);
-      }).catch(() => {
-        console.error('Failed to pull notifications!');
+  start(interval) {
+    if (interval === undefined) {
+      const args = {
+        data: {
+          url: 'http://localhost:5727/notification',
+          headers: {},
+        },
+        headers: { 'Content-Type': 'application/json' },
+      };
+      const setCallback = this.client.put(`${this.config.host}/notification/callback`, args, () => {});
```

Comment:
Užklausą į remote serverį reiktų siųsti po to kai inicializuoji serverį, o ne prieš, nes gali įvykti race condition.